addons/website/views/snippets/s_product_catalog.xml

```diff
@@ -48,26 +48,36 @@
 </template>
 
 <template id="s_product_catalog_dish">
-    <div class="s_product_catalog_dish col-lg-12 pt16" data-name="Product">
-        <div class="s_product_catalog_dish_title d-flex align-items-baseline border-bottom mb8">
-            <h5 t-esc="name"/>
-            <span class="ml-auto pl-3 h5" t-esc="price"/>
-        </div>
-        <p class="s_product_catalog_dish_description">Add a description here</p>
+    <div class="s_product_catalog_dish col-lg-12" data-name="Product">
+        <h6 class="s_product_catalog_dish_title d-flex align-items-baseline mb-0 py-1">
+            <span t-esc="name" class="s_product_catalog_dish_name"/>
+            <span t-esc="price" class="s_product_catalog_dish_price"/>
+        </h6>
```

Comment:
This is what I thought at the beginning. But what is the alternative? A paragraph? Spans and divs are messing up the layout when using the font style selector (normal, h1, h2...).

Reply:
Sure it does not work with divs ? But I think that ideally, it should be a `<li/>` instead a `<ul/>`, this is what this DOM really is: a list. I think that bootstrap have a class to remove all associated browser styles of those and simply use them as a structure. And I think the editor would maybe work better with `<li/>`

Reply:
Had the same idea. Already did that change. But this is not helping as it intervenes at the parents level. Because a list item have to include the name, the price and the description.
There is no perfect solution here.